governance/governance.py

```diff
@@ -339,7 +344,7 @@ def setStepPrice(self, stepPrice: int):
             self.StepPriceChanged(stepPrice)
 
     @external
-    def acceptScore(self, txHash: bytes):
+    def acceptScore(self, txHash: bytes, warning: str =""):
```

Comment:
minor thing but it's better to add a space between punctuations.  
```suggestion
    def acceptScore(self, txHash: bytes, warning: str = ""):
```